tests/tests_clusters/test_cluster_lifecycle.py

```diff
@@ -0,0 +1,53 @@
+import os
+import uuid
+
+import pytest
+
+from src.lightning_app.testing.testing import run_cli
+
+
+@pytest.mark.cloud
+@pytest.mark.skipif(
+    os.environ.get("LIGHTNING_BYOC_ROLE_ARN") is None, reason="missing LIGHTNING_BYOC_ROLE_ARN environment variable"
+)
+@pytest.mark.skipif(
+    os.environ.get("LIGHTNING_BYOC_EXTERNAL_ID") is None,
+    reason="missing LIGHTNING_BYOC_EXTERNAL_ID environment variable",
+)
+def test_cluster_lifecycle() -> None:
```

Comment:
This PR introduced this new tests_clusters directory. But which testing job runs it?

Reply:
These aren't hooked up to our CI today because during manual verification cluster creation took more than one hour. There's an internal ticket tracking hooking this up to a CI - GRID-9821.

Reply:
Is there a reason why these tests are maintained outside the app folder?

Reply:
@awaelchli there is - they don't execute any apps and don't depend on any apps.

Reply:
I would suggest removing this file for now until they actually get hooked into CI. Not only they can silently break, they also give a false sense of security to unaware contributors.

Reply:
let me talk to @Borda next week about this; as this is part of our cloud offering we might be able to shift them to the non-public part of our codebase.